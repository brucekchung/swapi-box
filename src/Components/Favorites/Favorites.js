import React from 'react'

const Favorites = () => (
  <div>
    Favorites container
  </div>
)

export default Favorites