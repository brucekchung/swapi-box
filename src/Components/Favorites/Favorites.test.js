import React from 'react'
import { shallow } from 'enzyme'
import Favorites from './Favorites'

describe('Favorites', () => {
  const props = {
    favoriteData: [{
      "model": "Digger Crawler", 
      "name": "Sand Crawler",
    }],
    favorite: () => {},
  }

  it('should match the snapshot', () => {
    const wrapper = shallow(<Favorites {...props}/>)
    
    expect(wrapper).toMatchSnapshot()
  })
})