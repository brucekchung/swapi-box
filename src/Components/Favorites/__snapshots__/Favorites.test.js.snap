// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Favorites
    favorite={[Function]}
    favoriteData={
        Array [
            Object {
              "model": "Digger Crawler",
              "name": "Sand Crawler",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Card
          data={
                    Object {
                              "model": "Digger Crawler",
                              "name": "Sand Crawler",
                            }
          }
          favorite={[Function]}
          type="favoriteCard"
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "favorite0",
        "nodeType": "function",
        "props": Object {
          "data": Object {
            "model": "Digger Crawler",
            "name": "Sand Crawler",
          },
          "favorite": [Function],
          "type": "favoriteCard",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Card
            data={
                        Object {
                                    "model": "Digger Crawler",
                                    "name": "Sand Crawler",
                                  }
            }
            favorite={[Function]}
            type="favoriteCard"
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "favorite0",
          "nodeType": "function",
          "props": Object {
            "data": Object {
              "model": "Digger Crawler",
              "name": "Sand Crawler",
            },
            "favorite": [Function],
            "type": "favoriteCard",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
